src/mame/drivers/ksys573.cpp

```diff
@@ -2250,6 +2423,8 @@ void ksys573_state::konami573(machine_config &config)
 
 	adc0834_device &adc(ADC0834(config, "adc0834"));
 	adc.set_input_callback(FUNC(ksys573_state::analogue_inputs_callback));
+
+	JVS_MASTER(config, "jvs_master", 0);
```

Comment:
You can use object finders in place of literal tags to make things less error-prone.